crates/fuel-core/src/graphql_api/prometheus.rs

```diff
@@ -45,41 +35,33 @@ impl Extension for PrometheusExtInner {
     ) -> Response {
         let start_time = Instant::now();
         let result = next.run(ctx).await;
-
-        let op_name = self.operation_name.read().await;
-        if let Some(op) = &*op_name {
-            GRAPHQL_METRICS.graphql_observe(op, start_time.elapsed().as_secs_f64());
-        }
+        let seconds = start_time.elapsed().as_secs_f64();
+        GRAPHQL_METRICS.graphql_observe("request", seconds);
```

Comment:
do we still need this if the resolve method is what actually logs the ops?

Reply:
I prefer to keep it because, in this case, we can see how long someone has waited for some request. Because one request may have 100+ resolves. And based on information from it we can start future investigations